### PR TITLE
Fix unneeded imports

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
Feel free to squash/merge and delete the branch if you're happy

File: test/escape_test.dart [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'package:matcher/src/util.dart';
-import 'package:test/test.dart';
+import 'package:test/test.dart' show group, test, expect, isTrue;
```
Comment (jakemac53) [inline test/escape_test.dart:6]: Should we not just remove the `matcher` import?

Comment (kevmoo) [reply test/escape_test.dart:6]: I know it seems a bit academi. Trying to make sure we're really getting it from matcher. 🤷

Comment (jakemac53) [reply test/escape_test.dart:6]: Why is that important here?

**Comment by kevmoo** [comment]
@jakemac53 – PTAL

**Review by jakemac53 — APPROVED** [review]

**Pull request merged**